---
result: FAILURE
url: http://wilson.ci.chef.co/job/chef-analytics-trigger-ad_hoc/45/
failure_category: unknown
failure_cause: 'unknown: all: chef-analytics-test'
timestamp: 2016-02-10 23:37:08 UTC
duration: 50m48s
triggered_by: kmacgugan
active_duration: 50m35s
parameters:
  GIT_REF: kyleen/spool-61
  EXPIRE_CACHE: false
change:
  git_remote: git@github.com:chef/chef-analytics.git
  git_commit: 5f43d79d642faabaae0d8eb1b6bb1b2572714fef
  project: opscode-analytics
  version: 1.2.1+20160210233730.git.12.5f43d79
stages:
  chef-analytics-test:
    result: FAILURE
    failure_category: unknown
    failure_cause: 'unknown: all'
    url: http://wilson.ci.chef.co/job/chef-analytics-test/119/
    duration: 24m12s
    runs:
      el-5:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=el-5,project=opscode-analytics,role=tester/119/
        duration: 19m55s
        logs:
          consoleText:
            3323: |
              Webhook Smoke
                when a new rule is created
              [2016-02-10T19:22:50-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:22:55-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:23:00-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:23:05-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:23:10-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-10T19:23:15-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:23:20-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:23:25-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:23:30-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:23:35-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            3411: |
              Randomized with seed 15741

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 19m55s
          before (setup time): 8m16s
          chef-client private-chef::default: 2m57s
          chef-client opscode-analytics::default: 3m44s
          after (cleanup time): 2m56s
      el-6:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=el-6,project=opscode-analytics,role=tester/119/
        duration: 22m6s
        logs:
          consoleText:
            3300: |
              Webhook Smoke
                when a new rule is created
              [2016-02-10T19:27:14-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:27:19-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:27:24-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:27:29-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:27:34-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-10T19:27:40-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:27:45-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:27:50-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:27:55-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:28:00-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            3388: |
              Randomized with seed 45441

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 22m6s
          before (setup time): 8m27s
          chef-client private-chef::default: 5m26s
          chef-client opscode-analytics::default: 5m27s
          after (cleanup time): 42s
      el-7:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=el-7,project=opscode-analytics,role=tester/119/
        duration: 24m11s
        logs:
          consoleText:
            33403: |
              warning: file /opt/opscode-analytics/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.1/ri/Nokogiri/XML/SyntaxError/int1-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.1/ri/Nokogiri/XML/SyntaxError/file-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.1/ri/Nokogiri/XML/SyntaxError/fatal%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.1/ri/Nokogiri/XML/SyntaxError/error%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.1/ri/Nokogiri/XML/SyntaxError/domain-i.ri: remove failed: No such file or directory
            45526: |
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception: remove failed: No such file or directory
            46469: |
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/install.json: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/install.json: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/MYMETA.json: remove failed: No such file or directory
            46848: |
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/Test/File.pm: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/Test/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/Test/Exception.pm: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/site_perl/5.18.1/Test/Dir/Base.pm: remove failed: No such file or directory
            50661: |
              warning: file /opt/opscode-analytics/embedded/lib/perl5/5.18.1/File/Basename.pm: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/5.18.1/File: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/5.18.1/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/5.18.1/ExtUtils/xsubpp: remove failed: No such file or directory
              warning: file /opt/opscode-analytics/embedded/lib/perl5/5.18.1/ExtUtils/typemap: remove failed: No such file or directory
            113745: |
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/int1-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/file-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/fatal%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/error%3f-i.ri: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/ruby/gems/2.1.0/doc/nokogiri-1.6.3.1/ri/Nokogiri/XML/SyntaxError/domain-i.ri: remove failed: No such file or directory
            122793: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Fatal: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception/.packlist: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/auto/Test/Exception: remove failed: No such file or directory
            123739: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-File-1.44: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014/MYMETA.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Fatal-0.014: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/install.json: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/x86_64-linux-thread-multi/.meta/Test-Exception-0.43/MYMETA.json: remove failed: No such file or directory
            124119: |
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Exception.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/site_perl/5.18.1/Test/Dir/Base.pm: remove failed: No such file or directory
            127932: |
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File/Basename.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/File: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/Fatal.pm: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/xsubpp: remove failed: No such file or directory
              warning: file /opt/opscode/embedded/lib/perl5/5.18.1/ExtUtils/typemap: remove failed: No such file or directory
            139869: |
              Webhook Smoke
                when a new rule is created
              [2016-02-10T19:27:04-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:27:09-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:27:14-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:27:19-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:27:24-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-10T19:27:29-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-10T19:27:34-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-10T19:27:39-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-10T19:27:44-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-10T19:27:49-05:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            139957: |
              Randomized with seed 43690

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 24m11s
          before (setup time): 10m13s
          chef-client private-chef::default: 7m13s
          chef-client opscode-analytics::default: 1m31s
          after (cleanup time): 3m6s
      ubuntu-10.04:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=ubuntu-10.04,project=opscode-analytics,role=tester/119/
        duration: 20m44s
        logs:
          consoleText:
            3357: |
              Webhook Smoke
                when a new rule is created
              [2016-02-11T00:23:37+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:23:42+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:23:47+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:23:52+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:23:57+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-11T00:24:02+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:24:07+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:24:12+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:24:17+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:24:22+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            3445: |
              Randomized with seed 31158

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 20m44s
          before (setup time): 7m23s
          chef-client private-chef::default: 5m33s
          chef-client opscode-analytics::default: 2m24s
          after (cleanup time): 3m16s
      ubuntu-12.04:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=ubuntu-12.04,project=opscode-analytics,role=tester/119/
        duration: 16m5s
        logs:
          consoleText:
            3352: |
              Webhook Smoke
                when a new rule is created
              [2016-02-11T00:18:59+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:19:04+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:19:09+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:19:14+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:19:19+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-11T00:19:24+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:19:29+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:19:34+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:19:39+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:19:45+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            3440: |
              Randomized with seed 62343

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 16m5s
          before (setup time): 7m45s
          chef-client private-chef::default: 1m57s
          chef-client opscode-analytics::default: 1m24s
          after (cleanup time): 2m56s
      ubuntu-14.04:
        result: FAILURE
        failure_category: unknown
        failure_cause: unknown
        url: http://wilson.ci.chef.co/job/chef-analytics-test/architecture=x86_64,platform=ubuntu-14.04,project=opscode-analytics,role=tester/119/
        duration: 23m36s
        logs:
          consoleText:
            3410: |
              Webhook Smoke
                when a new rule is created
              [2016-02-11T00:26:28+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:26:33+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:26:39+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:26:44+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:26:49+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5
                  it creates a webhook rule
                    has a successful code (FAILED - 1)
                    returns the rule location (FAILED - 2)
                  when a chef event is fired
                    receives a chef event (FAILED - 3)
              [2016-02-11T00:26:54+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 1/5
              [2016-02-11T00:26:59+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 2/5
              [2016-02-11T00:27:04+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 3/5
              [2016-02-11T00:27:09+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 4/5
              [2016-02-11T00:27:14+00:00] ERROR: Error connecting to https://manage.chef.piab/id/oauth/token?grant_type=password, retry 5/5

              An error occurred in an `after(:context)` hook.
            3498: |
              Randomized with seed 24185

              Build step 'Invoke XShell command' marked build as failure
              Performing Post build task...
              Match found for : : True
        steps:
          total: 23m36s
          before (setup time): 7m30s
          chef-client private-chef::default: 6m55s
          chef-client opscode-analytics::default: 3m51s
          after (cleanup time): 3m13s
  chef-analytics-build:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-analytics-build/135/
    duration: 26m8s
    runs:
      el-5:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-analytics-build/architecture=x86_64,platform=el-5,project=opscode-analytics,role=builder/135/
        duration: 25m52s
      el-6:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-analytics-build/architecture=x86_64,platform=el-6,project=opscode-analytics,role=builder/135/
        duration: 19m4s
      el-7:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-analytics-build/architecture=x86_64,platform=el-7,project=opscode-analytics,role=builder/135/
        duration: 24m38s
      ubuntu-10.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-analytics-build/architecture=x86_64,platform=ubuntu-10.04,project=opscode-analytics,role=builder/135/
        duration: 17m40s
      ubuntu-12.04:
        result: SUCCESS
        url: http://wilson.ci.chef.co/job/chef-analytics-build/architecture=x86_64,platform=ubuntu-12.04,project=opscode-analytics,role=builder/135/
        duration: 17m23s
  chef-analytics-trigger-ad_hoc:
    result: SUCCESS
    url: http://wilson.ci.chef.co/job/chef-analytics-trigger-ad_hoc/45/
    duration: 15s
successful_logs_analyzed: false
